android/.gitignore

```diff
@@ -5,5 +5,9 @@ gradle-wrapper.jar
 /gradlew.bat
 /local.properties
 GeneratedPluginRegistrant.java
+
+# Remember to never publicly share your keystore.
+# See https://flutter.dev/docs/deployment/android#reference-the-keystore-from-the-app
 key.properties
-*.jks
\ No newline at end of file
+**/*.keystore
+# **/*.jks # FIXME: CI depends on `android/tat.jks`
```

Comment:
Suggest stating the actual reason.
Only writing 'CI depends on tat.jks' does not provide the next person who sees the 'FIXME' with more information about the expected improvement or why we think it is not a good practice.

Therefore, I recommend changing it to

```suggestion
# **/*.jks # FIXME: Revise the CI build process to dynamically load the tmp jks file from secret, rather than including unnecessary testing jks file in the project.
```

Reply:
Good suggestion. I didn't know what exactly was the root cause at that time. I think "Revise the CI build process to dynamically load the temporary JKS file from a secret, rather than including an unnecessary testing JKS file in the project." would be clearer.